src/components/beta-tag/BetaTag.tsx

```diff
@@ -1,27 +1,26 @@
-import {StyleProp, View, ViewStyle} from 'react-native';
-import {ThemeText} from '@atb/components/text';
+import {StyleProp, ViewStyle} from 'react-native';
 import {StyleSheet} from '@atb/theme';
 import React from 'react';
+import {InfoChip} from '../info-chip';
 
 type Props = {
   style?: StyleProp<ViewStyle>;
 };
 export const BetaTag = ({style}: Props) => {
   const styles = useStyles();
   return (
-    <View style={[styles.betaLabel, style]}>
-      <ThemeText type={'body__tertiary'} color="background_accent_2">
-        Beta
-      </ThemeText>
-    </View>
+    <InfoChip
```

Comment:
I believe this still kinda defeats the purpose. It's still a one-off UI component (has explicitly set backgroundColor and borderRadius)

Reply:
Explicitly setting backgroundColor and borderRadius is currently needed to match the exact Figma drawings as far as I can tell?

Reply:
Well, then it's the Figma drawings that are incorrect. I sincerely don't believe the designers intended to introduce a new UI element just for this single purpose. If for some reason this is the case, the `BetaTag` component should be moved to the `components` folder.

Reply:
I suggest you check this with the designers.

Reply:
Following discussion with the designers, it should be kept as is and moved to the components folder. Perhaps it should then also receive text as input, and be called InfoTag instead?

Reply:
@KristianSelnas done